pipelines/qc/Snakefile

```diff
@@ -1,36 +1,38 @@
 import os
 import numpy as np
 import pandas as pd
+import re
 
-sampleset = os.path.split(os.getcwd())[1]
 if os.path.isfile("configuration.yml"):
   os.remove("configuration.yml")
 
 # read_fofn
-fofn_path = os.path.join(config["vector_ops"], "tracking", sampleset, "agam.fofn.tsv")
+pat = re.compile("^.+(AG1000G-\D{2}(-\D)?)/agam.fofn.tsv$")
+fofn_path = config["fofn"]
+mm = pat.match(fofn_path)
+assert mm is not None, "No sampleset found in path {!r}".format(fofn_path)
+sampleset = mm.group(1)
+
 fofn = pd.read_table(fofn_path)
 samples = fofn['sample'].unique()
 
-# set script dir
-script_dir = "../../scripts"
-
 # create a table of paths
 path_df = pd.DataFrame(index=samples)
 path_df["path"] = [os.path.join(config["input_location"], x + ".zarr.zip") for x in samples]
 
 # HACK NOT IN PRODUCTION>>>
 path_df["exists"] = path_df.path.map(os.path.isfile)
 samples = path_df.loc[path_df.exists].index.tolist()
-print(len(samples))
 
 
 rule qc_report:
   input:
-    manifest="manifest",
-    csv=expand("{sample}.callstats.csv", sample=samples),
-    npy=expand("{sample}.covhist.npz", sample=samples),
-    contam=expand("{sample}.contamination.csv", sample=samples),
-    cfg="configuration.yml"
+    manifest=os.path.join(sampleset, "manifest"),
```

Comment:
Purely stylistic remark: Snakemake encourages using `"{sampleset}/manifest"` in such cases. I'm not sure, but probably you can set a global wildcard by inserting 
```
wildcard_constraints:
    sampleset=sampleset
```
which should set wildcard `sampleset` constraint to string value of `sampleset`

Reply:
Also applicable to further `os.path.join`